src/main/java/com/CezaryZal/api/body/manager/BodySizeService.java

```diff
@@ -0,0 +1,58 @@
+package com.CezaryZal.api.body.manager;
+
+import com.CezaryZal.api.body.BodySizeRepository;
+import com.CezaryZal.api.body.entity.BodySize;
+import com.CezaryZal.api.body.entity.BodySizeDao;
+import com.CezaryZal.api.body.entity.BodySizeDto;
+import com.CezaryZal.api.body.manager.mapper.ConverterBodySizeToDto;
+import com.CezaryZal.api.body.manager.mapper.ConverterDaoToBodySize;
+import com.CezaryZal.api.body.manager.repo.BodySizeRepoService;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Service
+public class BodySizeService extends BodySizeRepoService {
+
+    private ConverterBodySizeToDto converterBodySizeToDto;
+    private ConverterDaoToBodySize converterDaoToBodySize;
+
+    public BodySizeService(BodySizeRepository BSRepository,
+                           ConverterBodySizeToDto converterBodySizeToDto,
+                           ConverterDaoToBodySize converterDaoToBodySize) {
+        super(BSRepository);
+        this.converterBodySizeToDto = converterBodySizeToDto;
+        this.converterDaoToBodySize = converterDaoToBodySize;
+    }
+
+    public BodySizeDto getBodySizeDtoById(Long id) {
+        return converterBodySizeToDto.mappingEntity(getBodyById(id));
+    }
+
+    public BodySizeDto getBodyDtoByDateAndUserId(String inputDate, Long userId) {
+        return converterBodySizeToDto.mappingEntity(getBodyByDateAndUserId(inputDate, userId));
+    }
+
+    public List<BodySizeDto> getListBodySizeDto() {
+        List<BodySize> allBodySize = getAll();
+        List<BodySizeDto> listBodySizeDto = new ArrayList<>();
+        allBodySize.forEach(tmpBS -> listBodySizeDto.add(converterBodySizeToDto.mappingEntity(tmpBS)));
```

Comment:
List<BodySizeDto> collect = allBodySize.stream()
                .map(tmpBS -> converterBodySizeToDto.mappingEntity(tmpBS))
                .collect(Collectors.toList());

Reply:
nad tym się zastanawiałem jak lepiej to zmapować.
Ok dzięki :)